README.md

```diff
@@ -1,15 +1,111 @@
+  the total card count is 52.
+* Human player always starts the game first.
+* When the Draw two action card is played, the next player automatically loses his turn, gets two cards added to his current set 
+  set of cards and the current player can choose the color to continue the game (same as a normal UNO game).
+* When the draw one action card is played, the next player by default loses his turn and a card will be added to his set of cards.
+
+
+## Group Information
+
+**Organization:** Carleton Unviersity (SYSC 5709)
+
+### Authors:
+
+**1. Mia Wei**
+ - GitHub: MiaYWei
+
+**2. Krithiga Subramaniam Ravichandran**
+
+ - GitHub: krithigasr
+
+**3. Hongbo Pang**
+
+ - GitHub: HongBo713
+
+**4. Aditi Bhardwaj**
+
+ - GitHub: aditibhardwaj98
+
+## Brief Description
+Following are the rules with which the game has been designed
+
+1. There are 52 cards in total - 40 normal cards, 12 action cards.
+2. These are 10 cards in each color - red,green,yellow and blue and action cards are - 4 Draw One cards, 4 Skip cards, 2 Wild cards,
+   2 Wild Draw two cards.
+3. To be declared as the winner of the game, you have to be the first player to exhaust all the cards assigned to you.
+4. During the start of the game, 5 random cards are assigned to each player - human player and the computer, in this case.
+5. One card is placed on the table with which you have to start matching your cards.
+6. To keep things simple, the human player always starts the game first.
+7. These are the keyboard inputs with which human can play the game
+   * To request a card from the Draw pile, Press N or n.
+   * To end turn, Press E or e.
+   * To discard a normal card - Say you want to discard(RED,3), Press R3.
+   * To discard action cards, 
+     * Skip cards- Combine the first letter of the color with 'S', so if you want to discard(RED,SKIP), Press RS.
+     * Draw one cards- Combine the first letter of the color with 'O', so if you want to discard(RED,DRAW-ONE), Press R0.
+     * Wild cards- Press AW.
+     * Wild cards- Press AT.
+     (A above stands for Action cards, W stands for Wild, T stands for draw Two)
+   * To quit the game, Press Q or q. (This can happen at any point of time).
+
+
+## File Structure
+
+The following file structure is used to organize files in this repository:
+
+* doc - Includes the designed flowhcart for the application.  The main documentation can be found on the [UNO Express Card Game Documentation](https://github.com/MiaYWei/Group_G_UNO-Card-Game/wiki/User-Manual).
+* include - The main header files are listed below:
+  *  All Header files are included here that are used throughout the application.
+* src - Contains the definition for all functions.
+* test
+  * include - Contains the declarations for all unit tests.
+    * Includes the header files for the test files.       
+  * src - Contains the definitions for all unit tests.  
+    * test.c - Main file that calls the test functions in other files.
```

Comment:
Can we make it similar to:
![image](https://user-images.githubusercontent.com/33041048/102627348-1443b680-4116-11eb-8707-0cb8cd4231f6.png)

Reply:
Do you want to explain all the folders?